manifests/init.pp

```diff
@@ -1,6 +1,7 @@
 # A module to manage S3 mounts using yas3fs
 class yas3fs (
   $install_pip_package = $yas3fs::params::install_pip_package,
+  $install_init        = $yas3fs::params::install_init,
```

Comment:
Is this the best name for the parameter? Since we use `install_pip_package` as a boolean parameter to install pip, I can't help but wonder if this implies that we're installing init. I'm not sure what it should be called though - maybe something like `init_system`?